tests/sympc/grad/grad_functions_test.py

```diff
@@ -814,3 +815,55 @@ def test_grad_maxpool_2d_backward(get_clients, kernel_size, stride, padding) ->
 
     res = res_mpc.reconstruct()
     assert np.allclose(res, expected_grad, rtol=1e-3)
+
+
+def test_grad_div_forward(get_clients) -> None:
+    # We need Function Secret Sharing (only for 2 parties) for
+    # comparing
+    parties = get_clients(2)
+    x = torch.tensor([2.0, 3.0])
+    y = torch.tensor([6.0, 4.0])
+
+    x_mpc = x.share(parties=parties)
+    y_mpc = y.share(parties=parties)
+
+    ctx = {}
+    res_mpc = GradDiv.forward(ctx, x_mpc, y_mpc)
+
+    res = res_mpc.reconstruct()
+    expected = x / y
+
+    assert np.allclose(res, expected, rtol=1e-3)
+
+
+def test_grad_div_backward(get_clients) -> None:
+    parties = get_clients(2)
+
+    session = Session(parties=parties)
+    session.autograd_active = True
+    SessionManager.setup_mpc(session)
+
+    x_secret = torch.tensor([1.0, 2.0, 3.0, 4.0], requires_grad=True)
+    x = MPCTensor(secret=x_secret, session=session, requires_grad=True)
+
+    y_secret = torch.tensor([2.0, 3.0, 4.0, 5.0], requires_grad=True)
+    y = MPCTensor(secret=y_secret, session=session, requires_grad=True)
+
+    z = (x_secret / y_secret).sum()
```

Comment:
OUGHTTO: Can you remove the `sum` from here?
For the moment this test should check only the `div` functionality (you can pass in the gradient when doing `backward`)

Reply:
Done